crates/apollo-mcp-server/src/schema_tree_shake.rs

```diff
@@ -11,7 +11,7 @@ use apollo_compiler::validation::WithErrors;
 use apollo_compiler::{Name, Node, Schema};
 use std::collections::HashMap;
 
-struct RootOperationNames {
+pub struct RootOperationNames {
```

Comment:
I think we don't need to add pub to this, `new` or `name_for_operation_type` anymore, it might just been from our earlier testing?